src/components/twilio-voice-basic-call-control/routes.js

```diff
@@ -1,11 +1,40 @@
 import { Router } from 'express';
 import Twilio from 'twilio';
 import config from '../../config.js';
+import { isPhoneNumber } from '../../common/utils.js';
+import { getToken, postConferenceEvents, postTwiml } from '../../common/routes.js';
 
 const router = Router();
-const { accountSid, apiKeySid, apiKeySecret } = config;
+const { accountSid, apiKeySid, apiKeySecret, authToken, callerId } = config;
 const client = Twilio(apiKeySid, apiKeySecret, { accountSid });
 
+// Add your own authentication mechanism here to make sure this endpoint is only accessible to authorized users.
+router.get('/token', (req, res) => getToken({ req, res }));
+
+// Validate incoming Twilio requests
+// https://www.twilio.com/docs/usage/tutorials/how-to-secure-your-express-app-by-validating-incoming-twilio-requests
+router.post('/twiml', Twilio.webhook({ protocol: 'https' }, authToken), (req, res) =>
+  postTwiml({
+    req,
+    res,
+    caller: {
+      label: 'agent1',
+      // 3 participants needed for warm transfer: agent1, agent2, and customer.
+      maxParticipants: 3,
+      // Allow agent1 to drop from call, while agent2 and customer continue call.
+      endConferenceOnExit: false,
+      componentUrl: 'twilio-voice-basic-call-control',
+    },
+    callee: { label: 'customer' },
```

Comment:
Can we flatten this out? It seems the rest of the options are not caller/callee specific.
```
{
  callerLabel,
  calleeLabel,
  maxParticipants,
  endConferenceOnExit,
  componentUrl,
}
```

Reply:
After flattening out, I think its easier to read if we keep the params wrapped in an object. wdyt?